ui/window.go

```diff
@@ -59,6 +61,7 @@ func CreateWindow(wal *wallet.Wallet) (*Window, error) {
 	win.tabs.Flex.Spacing = layout.SpaceBetween
 	win.current = win.WalletsPage
 	win.dialog = func() {}
+	win.sendPage = win.NewSendPage(win.theme)
```

Comment:
You shouldn't need the "sendPage" in the Window struct. Also, if "NewSpendPage" is a method on "Window", there's no need to pass "win.theme" as an argument to it.